.github/workflows/docker-build-publish.yml

```diff
@@ -0,0 +1,123 @@
+name: Docker Build & Publish
+
+on:
+  push:
+    branches:
+      - "**"
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]+-alpha.[0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]+-beta.[0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+"
+  pull_request:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      version:
+        # Friendly description to be shown in the UI instead of 'name'
+        description: "Semver type of new version (major / minor / patch)"
+        # Input has to be provided for the workflow to run
+        required: true
+        type: choice
+        options:
+          - patch
+          - minor
+          - major
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+  MAINTAINER: CelestiaOrg
+  DESCRIPTION: CelestiaOrg repository ${{ github.repository }}
+
+jobs:
+  docker-security:
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: Checkout
+        uses: "actions/checkout@v3"
+
+      - name: Build and Push
+        uses: docker/build-push-action@v3
+        with:
+          push: false
+          platforms: linux/amd64
+          tags: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:latest
```

Comment:
If I understand this correctly, we need to identify a tag in order to reference it in the next step correct?

Since we are not pushing anything, this tag is only every used as a local reference?

If that is the case, can you add a comment to document that? Maybe we could even change the tag to `local` or something to further avoid potential confusion about defining a `latest` image.

Reply:
Exactly @MSevey 
We're building the container prior the scan.
I like the idea to change the tag to `local` instead of `latest`👌